packages/victory-core/src/victory-label/victory-label.js

```diff
@@ -3,11 +3,11 @@ import React from "react";
 import PropTypes from "prop-types";
 import VictoryPortal from "../victory-portal/victory-portal";
 import Rect from "../victory-primitives/rect";
-import CustomPropTypes from "../victory-util/prop-types";
-import Helpers from "../victory-util/helpers";
-import LabelHelpers from "../victory-util/label-helpers";
-import Style from "../victory-util/style";
-import Log from "../victory-util/log";
+import * as CustomPropTypes from "../victory-util/prop-types";
+import * as Helpers from "../victory-util/helpers";
+import * as LabelHelpers from "../victory-util/label-helpers";
+import * as Style from "../victory-util/style";
+import * as Log from "../victory-util/log";
 import TextSize from "../victory-util/textsize";
```

Comment:
I'm still working on TextSize - this one broke some tests and I'm still determining whether this is an issue with sinon mocking or some other kind of bug. Storybook looks the same, though.

Reply:
@beccanelson If you're still spinning your wheels on this, feel free to just give me some reproduction instructions and I can take a look too!

Reply:
I figured it out - the earlier version of sinon we were using for test stubbing doesn't support module exports. Upgrading sinon seems to be doing the trick to fix the tests.